incubator/vault/values.yaml

```diff
@@ -194,11 +196,15 @@ vault:
   #   secret:
   #     secretName: some-secret
   liveness:
+    # DR requires Vault enterprise
+    drEnabled: false
     aliveIfUninitialized: true
     aliveIfSealed: true
     initialDelaySeconds: 30
     periodSeconds: 10
   readiness:
+    # DR requires Vault enterprise
+    drEnabled: false
```

Comment:
Similarly to the comment above, should we consider `readyIfDRSecondary` instead?